### PR TITLE
Basic hover actions

### PR DESCRIPTION
**Pull request by adamschroder**

### CODE REVIEW

**Comment by adamschroder** [comment]
@iamtimm @jkeussen i'd love your thoughts on this.

**Comment by iamtimm** [comment]
This is coo! I'd try and include it in an existing file if possible.

**Comment by adamschroder** [comment]
thanks @iamtimm. The reason I put it in its own file though is because we could do other "actions" here. Hover animations, different states, etc. Instead of `show-on-parent-hover` we could do something like `animate-expand-on-parent-hover` or something.

anyway, what file do you think it should go in?

**Comment by iamtimm** [comment]
@adamschroder Ahh I see, well in that case lets keep it in there then, otherwise I would put it in the general.css file.

**Comment by adamschroder** [comment]
I added some stuff here to show which direction I was heading. Lemme know if this is something we want to expand on

File: src/css/lib/colors.css [hunk 1]
```diff
@@ -33,6 +33,7 @@
 
 .<%= c %> { color: var(--<%= c %>) }
 .hover-<%= c %>:hover { color: var(--<%= c %>) }
+.hover-background-<%= c %>:hover { background-color: var(--<%= c %>) }
```
Comment (iamtimm) [inline src/css/lib/colors.css]: @adamschroder Maybe we should change this class name to .bg-hover. What do you think?

Comment (adamschroder) [inline src/css/lib/colors.css]: yea good call, one sec

File: src/css/lib/colors.css [hunk 2]
```diff
@@ -33,6 +33,7 @@
 
 .<%= c %> { color: var(--<%= c %>) }
 .hover-<%= c %>:hover { color: var(--<%= c %>) }
+.hover-bg-<%= c %>:hover { background-color: var(--<%= c %>) }
```
Comment (iamtimm) [inline src/css/lib/colors.css]: @adamschroder I would even switch it to bg-hover so it matches everything else. bg-blue, bg-faded-red etc.

Comment (adamschroder) [inline src/css/lib/colors.css]: yea, but then it doesn't match the hover- one

Comment (adamschroder) [inline src/css/lib/colors.css]: it will be hover-bg-faded etc

Comment (iamtimm) [inline src/css/lib/colors.css]: I guess you're right here, make more sense. :)

**Pull request merged**